fluentd/configs.d/filter/k8s_record_transform.conf

```diff
@@ -1,4 +1,24 @@
-<filter kubernetes.**>
+<filter kubernetes.journal.container**>
+  type record_transformer
+  enable_ruby
+  <record>
+    hostname ${(kubernetes_host rescue nil) || File.open('/etc/docker-hostname') { |f| f.readline }.rstrip}
+    # log is set if MESSAGE is JSON valued and has err or msg field
+    message ${record['MESSAGE'] || log}
+    version 1.2.0
```

Comment:
@richm I guess that is more of what I meant, its different than what we are publishing with our current configuration.  `version` is more just for a quick sanity check and would make it easier for verifying that logs are the same -- we could look at logs with version=1.2.0 and version=1.3.0 and ensure they are similar enough.